common/src/main/java/com/distkv/common/utils/RuntimeUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.distkv.common.utils;
+
+import java.util.function.Supplier;
+
+public class RuntimeUtil {
+  private static final int WAIT_INTERVAL_MS = 1000;
```

Comment:
This should be shorter I think, maybe 200 is enough.

Reply:
OK